src/overlays/actors/ovl_Magic_Fire/z_magic_fire.c

```diff
@@ -28,7 +44,251 @@ const ActorInit Magic_Fire_InitVars = {
     (ActorFunc)MagicFire_Draw,
 };
 
-static ColliderCylinderInit D_80B8AF50 = {
+static u64 sTexture[] = {
```

Comment:
This data should probably just go in its own file.

Reply:
Discussed this and decided there's no point in moving it to a separate file since it's temporary https://discord.com/channels/688807550715560050/688851317593997489/803735826319671316